test/token.test.ts

```diff
@@ -57,4 +58,10 @@ describe("Token Module", async () => {
       );
     }
   });
+  it("Should parse events properly", async () => {
```

Comment:
This test is not super clear. The name is useless too.

The test title should be `should return the correct list of holders and their balances`

The test should:

1. Transfer tokens to multiple wallets (at least 5 wallets)
2. Call the `getAllHolderBalances` method and check the balance of every single wallet that we expect the tokens to be sent to.